Dockerfile

```diff
@@ -1,25 +1,30 @@
 # Use imutable image tags rather than mutable tags (like ubuntu:18.04)
 FROM ubuntu:bionic-20200807
 
-ENV KUBE_VERSION=v1.20.6
-
-RUN mkdir /kubespray
```

Comment:
it is fine to remove this `mkdir` because WORKDIR specified directory will be created if not existing.